lib/advisor_web/authentication/local_strategy.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Local.Strategy do
+  use Ueberauth.Strategy
+
+  alias Ueberauth.Auth.Credentials
+  alias Ueberauth.Auth.Info
+
+  def handle_request!(conn) do
+    redirect!(conn, Ueberauth.Strategy.Helpers.callback_path(conn))
+  end
+
+  def info(conn) do
+    %Info{
+      urls: %{website: "8thlight.com"},
+      email: "rswanson@8thlight.com",
+      name: "Ron Swanson"
```

Comment:
There is no "Ron Swanson" at 8th Light!

Reply:
What! I am sure I've see this person in the office: 
![Ron Swanson](https://uproxx.files.wordpress.com/2016/01/ron-swanson.jpg)